look-and-feel/react/src/index.ts

```diff
@@ -33,3 +33,4 @@ export { Svg } from "./Svg";
 export { TabsClient as Tabs, Direction as TabsDirection } from "./Tabs/Tabs";
 export { Tag } from "./Tag";
 export { Title, TitleSize, type TitleLevel } from "./Title";
+export { Skeleton } from "./Skeleton/Skeleton";
```

Comment:
ne faudrait-il pas avoir un index.ts dans Skeleton/ afin d'avoir un export comme pour les autres composant : 
```suggestion
export { Skeleton } from "./Skeleton";
```

Reply:
je ne sais pas, je me suis calé sur la façon de faire des autres composants, sinon oui j'ai plutôt l'habitude de créer un fichier index.ts.
Mais selon moi, il faudrait que l'on note cela dans l'ADR et qu'on uniformise l'ensemble des composants
@JLou tu en penses quoi ?

Reply:
il me semble que j'avais cru lire que les index.ts ralentissez les builds, perso jamais eu problème, mais il faudrait en discuter en séance pour valider une pratique.